scripts/reinforcement_learning/rsl_rl/actor_critic_conv2d.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import torch
+import torch.nn as nn
+from torch.distributions import Normal
+
+from rsl_rl.modules.actor_critic import get_activation
+
+
+class ResidualBlock(nn.Module):
```

Comment:
Since this is RSL-RL library related, I would recommend making an MR there for this actor critic structure :)

Reply:
Yes, we can do that. Then I suppose this PR can only be merged after the RSL-RL accepts my changes.